CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1 (December 26, 2019)
```

Comment:
Please, use 1.0.1 version, like this:

## 1.0.1 (December 26, 2019)

* Update sailor version to 2.5.4

## 1.0.0 (November 15, 2019)

* Initial release